pkg/operator/ceph/config/network_test.go

```diff
@@ -50,19 +50,14 @@ func TestGenerateNetworkSettings(t *testing.T) {
 	assert.Error(t, err)
 
 	//
-	// TEST 2: single dedicated networks
+	// TEST 2: single dedicated network
```

Comment:
We actually want to test `cluster_network` alone too.

Reply:
Done